feature-libs/order/components/order-history/order-history.module.ts

```diff
@@ -40,6 +68,9 @@ import { OrderHistoryComponent } from './order-history.component';
         },
       },
     }),
+    provideDefaultConfigFactory(() =>
+      inject(MYACCOUNT_V2_ORDER) ? myAccountV2CmsMapping : {}
+    ),
```

Comment:
[MINOR]
For clarity of the code inside this module, what about moving out all the my-account-v2 related code to a separate module, e.g. `MyAccountV2OrderHistoryModule`? 
And in `OrderHistoryModule` just import the `MyAccountV2OrderHistoryModule`?